src/macros.jl

```diff
@@ -78,3 +78,54 @@ function set_controller_reach!(sym_model_contr, sym_model_sys, X_init, X_target)
 	end
 	println("\nset_controller_reach! terminated with success")
 end
+
+# "Set" assumes sym_model_contr is empty initially... May fail if not respected
+function set_controller_safe!(sym_model_contr, sym_model_sys, X_init, X_safe)
+	@assert X_init.grid_space == X_target.grid_space ==
```

Comment:
Uses `===` otherwise, it will be slow once we implement `==` between grid spaces